api/v1alpha1/shared_types.go

```diff
@@ -401,11 +402,17 @@ const (
 
 // KubernetesPodDisruptionBudgetSpec defines Kubernetes PodDisruptionBudget settings of Envoy Proxy Deployment.
 type KubernetesPodDisruptionBudgetSpec struct {
-	// MinAvailable specifies the minimum number of pods that must be available at all times during voluntary disruptions,
+	// MinAvailable specifies the minimum amount of pods (can be expressed as integers or as a percentage) that must be available at all times during voluntary disruptions,
 	// such as node drains or updates. This setting ensures that your envoy proxy maintains a certain level of availability
-	// and resilience during maintenance operations.
+	// and resilience during maintenance operations. Cannot be combined with maxUnavailable.
```

Comment:
can we use CEL expressions to enforce this ? also add some tests under `test/cel-validation` will be appreciated

Reply:
here's an example https://github.com/envoyproxy/gateway/blob/a58021e6b4ae79fc268249cffcab8e342b04171b/api/v1alpha1/ext_auth_types.go#L10

Reply:
Thanks for the example, added along with updated tests. FWIW it's not a requirement that PDBs have at least one or the other, just that they can't be mixed.